src/change.js

```diff
@@ -0,0 +1,11 @@
+import { taskSection, tasksArray } from './index.js'//eslint-disable-line
+
+export default function change() {
+  taskSection.addEventListener('change', (event) => {
+    if (event.target.className === 'checkBoxClass') {
+      tasksArray[event.target.id].status = !tasksArray[event.target.id].status;
+      localStorage.setItem('task', JSON.stringify(tasksArray));
+      event.target.parentElement.classList.toggle('checked');
+    }
+  });
+}
```

Comment:
- [ ] OPTIONAL: It is best practice that your models be on a different folder to keep them tidy 💪

Reply:
Thank you  @Meltrust  for your comment I will do it  😃